x-pack/plugins/fleet/public/plugin.ts

```diff
@@ -83,6 +93,9 @@ export interface FleetStart {
   authz: FleetAuthz;
   registerExtension: UIExtensionRegistrationCallback;
   isInitialized: () => Promise<true>;
+  getBulkAssets: (
```

Comment:
Great that it works, though `getBulkAssets` looks out of place here. 
We could check in kibana if there is a pattern used to provide hooks to other plugins.

Reply:
Nice! Yes I think just moving it under some structure would be good, e.g. `hooks: {epm: {getBulkAssets}}`

Reply:
I changed it to `hooks: {epm: {getBulkAssets}}` as you suggested, so in the future we'll have an existing structure to export other hooks if needed.